web/src/components/Upstream/components/UpstreamSelector.tsx

```diff
@@ -61,6 +60,9 @@ const Component: React.FC<Props> = ({ shouldUpdate, onChange, list = [], disable
             {item.name}
           </Select.Option>
         ))}
+        <Select.Option value='None'>{
```

Comment:
Hi, please add i18n here :)

Reply:
OK, but it is a value, will not directly display for the user.

Reply:
oh yes